examples/wandb_sweeps/README.md

```diff
@@ -0,0 +1,5 @@
+# W&B Sweeps
+
```

Comment:
Could you add the following here?

* The example is just one possible grid. Tell them to make your own for your own problem by editing... (then also add a comment to that file like `# TODO: change this line ...`)
* Similarly you may want to tell them about how to enable Bayesian search.
* Could you enclose the names of the scripts in local hyperref links like `[foo](bar)`? 
* Could you have a code block showing the relevant pieces invoked?

Reply:
In the README, in triple-back-quoted code blocks, show the commands the reader will execute to run this example. Like:

```
./step_1.py
./step_2.py
```

or whatever.